lib/nimble_template/addons/variants/phoenix/web/bootstrap.ex

```diff
@@ -0,0 +1,117 @@
+defmodule NimbleTemplate.Addons.Phoenix.Web.Bootstrap do
+  @moduledoc false
+
+  use NimbleTemplate.Addons.Addon
+
+  @impl true
+  def do_apply(%Project{} = project, opts) do
+    project
+    |> edit_files(opts)
+    |> copy_files()
+  end
+
+  defp edit_files(%Project{} = project, opts) do
+    project
+    |> edit_assets_package()
+    |> edit_app_js(opts)
+    |> edit_app_scss(opts)
+    |> edit_vendor_index(opts)
```

Comment:
we're missing the test for this behavior: import `bootstrap.scss` to `vendor/_index.scss`

Reply:
We have it @longnd 

![image](https://user-images.githubusercontent.com/11751745/156495277-e823cfaa-ea4a-4193-9580-96d64d560b0c.png)

Reply:
that's right. My bad. I re-read the test 🙈